src/models/QueryStateModel.ts

```diff
@@ -27,16 +27,35 @@ export interface IQueryStateExcludedAttribute {
 }
 
 /**
- * The QueryStateModel is a key->value store of the state of every component that can affect a query.<br/>
- * Component set values in this key -> value store, and listen to event triggered to react accordingly.<br/>
- * For example, when a query is launched, the searchbox will set the 'q' attribute, the pager will set the 'first' attribute, etc.<br/>
- * At the same time, this class will trigger the associated event when a value is modified.<br/>
- * eg : The user change the content of the searchbox, and submit a query. This will trigger the following events :<br/>
- * -- state:change:q (because the value of 'q' changed)</br>
- * -- state:change (because at least one value changed in the query state)<br/>
- * Component or external code could hook handler on those events : document.addEventListener('state:change:q', handler);<br/>
- * See : {@link Model}, as all the relevant method are exposed in the base class.<br/>
- * Optionally, the state can be persisted to the query string to allow browser history management : See {@link HistoryController}
+ * The `QueryStateModel` class is a key-value store which contains the current state of the Coveo JavaScript Search
+ * Framework components that can affect the query (see [State](https://developers.coveo.com/x/RYGfAQ)). This class
```

Comment:
Maybe I just need a coffee but I had trouble understanding that sentence because of "Coveo Javascript Search Framework". For me it makes the sentence heavy and it splits the sentence in two.